sshuttle/methods/pf.py

```diff
@@ -108,16 +111,21 @@ def pf_query_nat(family, proto, src_ip, src_port, dst_ip, dst_port):
     pnl.direction = PF_OUT
     pnl.af = family
     memmove(addressof(pnl.saddr), packed_src_ip, length)
-    pnl.sxport.port = socket.htons(src_port)
     memmove(addressof(pnl.daddr), packed_dst_ip, length)
-    pnl.dxport.port = socket.htons(dst_port)
+    if sys.platform == 'darwin':
+        pnl.sxport.port = socket.htons(src_port)
+        pnl.dxport.port = socket.htons(dst_port)
+    else:
+        pnl.sxport = socket.htons(src_port)
+        pnl.dxport = socket.htons(dst_port)
```

Comment:
What is the purpose of the above change? To me, it looks like pn1.sxport.port and pn1.sxport refer to the same thing, as `pf_state_xport` is a union.